### ISSUE TITLE
dblclick

### ISSUE THREAD

**Issue by 2b3pro** [furf/jquery-ui-touch-punch#25]
Does jui-touch-punch recognize double-tap? I can't seem to make it work when my element is bound to a "dblclick"? Thoughts?

**Comment by playtcg** [comment]
I was just about to log this myself. Confirmed that it is still an issue. 
Tried it on:
- Android tablet running Honeycomb w default browser.
- Android ICS phone (Galaxy Nexus) w Google Chrome mobile beta.

**Comment by mliesen** [comment]
I have the same problem, can also add Opera mobile to the list.

**Comment by ghost** [comment]
Also having this problem, it is just because he didn't write any code for the double click though. I'm going to see if I can figure out how to write that, i'll let you know if I get it.

**Comment by GFoley83** [comment]
:+1: Still an issue it seems. @mordocai Did you ever come up with a solution?

**Comment by GFoley83** [comment]
Related issues: #43 #46 #5

**Comment by ghost** [comment]
@GFoley83 I don't believe I did. I'm not doing web development at my current job, and I think I left the job this issue was related to before I had a chance to fix it. In any case, looks like if you can implement something similar to what this article says might work http://appcropolis.com/howto/implementing-doubletap-on-iphones-and-ipads/.
I might work on it if I get the itch later this week, but it'd be on personal time.

**Comment by Masood10** [comment]
anyone ever solved that issue?

**Comment by Shafqat-Ali** [comment]
i'm having the same problem of dbclick after using jquery.ui.touch-punch.
is there anyone who solved this issue?

**Comment by Masood10** [comment]
I could not solve it so I used angular hammer double tap function. 
http://embed.plnkr.co/3sJFrC/
https://www.npmjs.com/package/angular-hammer

**Comment by Shafqat-Ali** [comment]
> I could not solve it so I used angular hammer double tap function.
> http://embed.plnkr.co/3sJFrC/
> https://www.npmjs.com/package/angular-hammer

can i use this angular-hammer into a simple application where i didn't used angular js?

**Comment by mighty-phoenix** [comment]
Are there any updates on this issue?

**Comment by RWAP** [comment]
No idea if anyone has solved this one, especially as this original branch of touch punch has not been updated in years.

Have you tried my branch - https://github.com/RWAP/jquery-ui-touch-punch

I brought together various suggested changes over the years in there and it seems to be much better behaved.

I don't own a touch device however!

**Comment by mcanepa** [comment]
@RWAP I was in the need of getting dblclick working on mobile and your fork did the job. Thanks!
